pkg/cloudprovider/providers/vsphere/vclib/virtualmachine.go

```diff
@@ -188,9 +188,9 @@ func (vm *VirtualMachine) GetResourcePool(ctx context.Context) (*object.Resource
 	return object.NewResourcePool(vm.Client(), vmMoList[0].ResourcePool.Reference()), nil
 }
 
-// Exists checks if the VM exists.
-// Returns false if VM doesn't exist or VM is in powerOff state.
-func (vm *VirtualMachine) Exists(ctx context.Context) (bool, error) {
+// IsActive checks if the VM is active.
+// Returns true if VM is in poweredOn state.
+func (vm *VirtualMachine) IsActive(ctx context.Context) (bool, error) {
```

Comment:
Not related to the change made in the PR, but just came across this unwanted code block.
I think this is not required.

```
if vmMoList[0].Summary.Config.Template == false {
 		glog.Warningf("VM is not in %s state", ActivePowerState)
 	} else {
 		glog.Warningf("VM is a template")
}
```